### ISSUE TITLE
Midterm Review

### ISSUE THREAD

**Issue by kaysieyu** [dchernysheva/ORIE-4741#8]
This team is looking at how COVID-19 related news headlines impact stock prices for different sectors of the stock market. 

Things I Liked
1. Very well define ways they will avoid over and underfitting. Cross validation is a great method. I think monitoring features with low significance is a great idea to mitigate fitting issues, as less lagged models could be really interesting to see how they might affect your models. 
2. Preliminary analysis diagrams and models were very well explained. I felt like I had sufficient context to understand them and wasn't overwhelmed with any unnecessary information. 
3. There are clear action steps that will be taken to complete the project. 

Areas of Improvement
1. Highly recommend modeling stocks from different sectors differently. For example, you showed the Delta stock and Costco stock, but the model AR1 model when used on Costco was heavily overfitting. 
2. Introduce additional features! It could be interesting to see how worldwide factors impact these multinational corporations. 
3. Could be interesting if you went into more detail on how you imputed the missing data values? What type of regression and loss functions did you use?

Overall, fantastic job! Really interesting project idea and I'm curious to see what this team comes up with.
